src/middleSection.js

```diff
@@ -0,0 +1,128 @@
+import { domElement } from './domElements.js'
+import { template } from './templates.js'
+import { sortElements } from './sortElements.js'
+/**
+ *  This function will get the scrollwidth and clientwidth of the itemscontainer to display the arrow
+ */
+const sideArrowVisibility = () => {
+  const middleCardContainer = document.querySelector('.items-container')
+  const middleContainerWidth = middleCardContainer.clientWidth
+  const scrollContainerWidth = middleCardContainer.scrollWidth
+  const itemsContainer = document.querySelector('.items-container')
+  if (middleContainerWidth >= scrollContainerWidth) {
+    domElement.leftArrow.style.display = 'none'
+    domElement.rightArrow.style.display = 'none'
+    itemsContainer.style.justifyContent = 'space-around'
+  } else {
+    domElement.leftArrow.style.display = 'block'
+    domElement.rightArrow.style.display = 'block'
+    itemsContainer.style.justifyContent = 'start'
+  }
+}
+
+/**
+ * This function will get the value from spinner  and update the card numbers
+ */
+const updateMiddleCitiesScrollButtonVisibility = () => {
```

Comment:
As this function is not just updating the scroll button visiblity but also calling `changeMiddleSegment` function, let's name it something different